lua/starfall/libs_sv/wire.lua

```diff
@@ -318,6 +318,16 @@ function wire_library.adjustOutputs (names, types)
 		types[i] = newtype
 	end
 	ent._outputs = { names, types }
+
```

Comment:
Would this cause entity outputs from a previous wire.adjustOuputs to be kept if we wanted to remove them?

Also should line 320 go after this?

Reply:
No, it only affects the names: "wirelink" and "entity" wire outputs which are lowercase and only created by the Wire ToolGun, lowercase in/out are not allowed by StarfallEx.

Good catch for the line 320, I'll check that tomorrow.